### ISSUE TITLE
VS NuGet PMUI - Machine-wide package sources area should be vertically resize-able as well

### ISSUE THREAD

**Issue by karann-msft** [NuGet/Home#7560]
@dstrube1 commented on [Tue Oct 23 2018](https://github.com/NuGet/docs.microsoft.com-nuget/issues/1109)

When resizing the Options window, the Available package sources area resizes with it, but the Machine-wide package sources area only resizes horizontally. It'd be great if the Machine-wide package sources area could be resized vertically as well, like the Available package sources area.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 108d587b-429e-3459-5698-3b40e44f4c3e
* Version Independent ID: 76ad6a3f-4dde-0883-1df9-0f173f0ca7e7
* Content: [NuGet Package Manager UI Reference](https://docs.microsoft.com/en-us/nuget/tools/package-manager-ui?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(VS.ToolsOptionsPages.NuGet_Package_Manager.Package_Sources)%26rd%3Dtrue)
* Content Source: [docs/tools/Package-Manager-UI.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/tools/Package-Manager-UI.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

**Comment by donnie-msft** [comment]
This UI is now part of Unified Settings.
